doc/Software Setup.md

```diff
@@ -107,23 +107,28 @@ sudo update-alternatives --install /usr/bin/python python /usr/bin/python3 2
 After the above commands are entered, the system should default to using Python 3. The default version can be switched back and forth via the command: `sudo update-alternatives --config python`
 
 ### 6. Install the RotorHazard Server
+
 Install the RotorHazard server code under '/home/pi/' on the Raspberry Pi as follows:
 
 Go to the [Latest Release page](https://github.com/RotorHazard/RotorHazard/releases/latest) for the project and note the version code.
 
 In the commands below, replace the two occurrences of "1.2.3" with the current version code, and enter the commands using a terminal window:
-```
+
+```bash
 cd ~
 wget https://codeload.github.com/RotorHazard/RotorHazard/zip/v1.2.3 -O temp.zip
 unzip temp.zip
 mv RotorHazard-1.2.3 RotorHazard
 rm temp.zip
 ```
 
-Enter the commands below to install RotorHazard server dependencies (be patient, this may take a few minutes):
-```
+The next step is to create a virtual environment, to install the RotorHazard server dependencies (be patient, this may take a few minutes):
+
+```bash
 cd ~/RotorHazard/src/server
-sudo pip install -r requirements.txt
+python3 -m venv venv
```

Comment:
We're using "python" (not "python3") everywhere else in the doc; can this be "python -m venv venv"?  Will this work okay under Python 3.7?

Reply:
Yes of course, I thought rotorhazard still uses python3 everywhere. But `python -m venv venv` would also be my preference 😉